core/src/components/refresher/test/a11y/refresher.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import AxeBuilder from '@axe-core/playwright';
+import { expect } from '@playwright/test';
+import { configs, test, dragElementBy } from '@utils/test/playwright';
+
+configs({ directions: ['ltr'], modes: ['md'], themes: ['light', 'dark'] }).forEach(({ config, title }) => {
+  test.describe(title('refresher: a11y for ion-color()'), () => {
+    test('should not have accessibility violations', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-content>
+          <ion-refresher class="refresher-native" slot="fixed">
+            <ion-refresher-content></ion-refresher-content>
+          </ion-refresher>
+        </ion-content>
+      `,
+        config
+      );
+
+      const target = page.locator('body');
+
+      await page.waitForSelector('ion-refresher.hydrated', { state: 'attached' });
+
+      await dragElementBy(target, page, 0, 320, undefined, undefined, false);
+
+      await page.waitForSelector('.arrow-container', { state: 'attached' });
```

Comment:
Why is this needed?

Reply:
I wanted to verify that the arrow container was visible when dragging since gestures tend to be flaky. Should I remove it?

Reply:
Rather than check something that is dependent on the internal markup of the component, it might be better to check the state of the component: https://github.com/ionic-team/ionic-framework/blob/fc88613fefa019a3b695a2c6e10c85cd3ce79ae8/core/src/components/refresher/refresher.tsx#L814

For example, checking the `refresher-refreshing` class verifies that a pull to refresh gesture was completed and the component is in the process of refreshing.

Reply:
I'll have to use `refreshing-pulling` since I'm testing the contrast while it's being pulled.